docs/debug_api.md

```diff
@@ -0,0 +1,165 @@
+
+To access the API over `curl`:
+
+```sh
+curl http://127.0.0.1:11100/inspectUsers?access_token=AS_TOKEN_FROM_REGISTRATION_FILE
+```
+
+## Endpoints
+
+<!-- no toc -->
+- [`GET /inspectUsers?regex={userRegex}`](#get-inspectusersregexuserregex)
+- [`POST /killRoom`](#post-killroom)
+- [`POST /killUser`](#post-killuser)
+- [`POST /reapUsers`](#post-reapusers)
+- [`GET /irc/$domain/user/$user_id`](#get-ircdomainuseruser_id)
+- [`POST /irc/$domain/user/$user_id`](#post-ircdomainuseruser_id)
+
+
+### `GET /inspectUsers?regex={userRegex}`
+
+#### Request Parameters
+
+- `userRegex` A JS regex string which should match against MXIDs. E.g. `@foobar_.*:matrix.org`
+
+#### Example Response
+
+```js
+{
+  "users": {
+    "@Half-Shot:half-shot.uk": [
+      {
+        "channels": [
+          "#matrix"
+        ],
+        "dead": false,
+        "server": "chat.freenode.net",
+        "nick": "Half-Shot"
+      }
+    ]
+  }
+}
+```
+
+### `POST /killRoom`
+
+Stop a room from being bridged. This will remove IRC ghost users from the room
+and disconnect Matrix users from the channel.
+
+The [Admin Room](admin_room.html#unlink) features a less
+powerful version of this command.
+
+#### Request Body
+
+```json5
+{
+  "room_id": "!foo:bar", // The Matrix Room ID. Required.
+  "domain": "irc.foo.bar", // The IRC domain. Required.
+  "channel": "#evilcave", // The IRC channel. Required.
+  "leave_notice": true, // Should a notice be sent on unbridge. Default: true.
+  "remove_alias": true, // Should the room alias for the room be removed. Default: true.
+}
+```
+
+#### Response Body
+
+The response body will contain a JSON array of stages that were successful and failed as
+it's possible for this command to only be partially successful.
+
+```json5
+error: [],
+stages: [],
+```
+
+### `POST /killUser`
+
+This will kill a connection to IRC for a given user on all networks they are connected to.
+
+#### Request Body
+
+```js
+{
+  "user_id": "@foo:bar",
+  "reason": "Trust nobody"
+}
+```
+
+#### Response Body
+
+If a disconnection was successful, the bridge will emit "null". Otherwise it may emit an error
+message in plaintext.
+
+### `POST /reapUsers`
+
+This will kill multiple connections for users considered "idle". This is a powerful and 
+expensive operation and should be taken with care.
+
+Idleness is calculated by how long it has been since a user has sent a message/joined/left a room.
+This is calulated by whether the appservice bot or it's users have seen the user perform any actions 
+(i.e. left a IRC bridged room or sent a message). Due to limitations of Matrix, it is not possible to 
+discover "lurkers".
+
+#### Request Parameters
+
+- `server` is the servername you wish to disconnect users from. This is the key of 
+  your server configuration object in the config section.
+- `since` is the number of hours a user has been idle for to be considered `idle`.
```

Comment:
An integer, I assume? May be worth documenting that just in case :)

Reply:
It's a query parameter, so a string

Reply:
Right – I meant it as „are floats allowed?”

Reply:
ahhhhhhhhhhhh no, will update